be/src/olap/column_vector.h

```diff
@@ -233,6 +233,13 @@ class ArrayColumnVectorBatch : public ColumnVectorBatch {
      */
     void get_offset_by_length(size_t start_idx, size_t size);
 
+    // From `start_idx`, put `size` ordinals to _item_offsets
+    // Ex:
+    // original _item_offsets: 0 3 5 9; ordinals to be added: 100 105 111; size: 3; satart_idx: 3
+    // --> _item_offsets: 0 3 5 (9 + 100 - 100) (9 + 105 - 100) (9 + 111 - 100)
+    // _item_offsets becomes 0 3 5 9 14 20
+    void put_item_ordinal(segment_v2::ordinal_t* ordinals, size_t start_idx, size_t size);
```

Comment:
does push back is enough? it seems we always append ordinals at the end.

Reply:
yes, it's enough we just need to caculate the ordinals read from _offset_iterator, and then put them into the back of ArrayColumnVectorBatch's _offsets.